addons/website/static/src/scss/website.ui.scss

```diff
@@ -527,3 +527,8 @@ body .modal {
         max-height: 14.2rem;
     }
 }
+
+// Post Submit Links
+.post_link:not(.o_post_link_js_loaded) {
+    pointer-events: none;
```

Comment:
If one receive the new css but not the new js, it will remain none forever, is that even possible / something to consider?

Reply:
> receive the new css but not the new js

Probably not possible, and I would not consider it anyway. The opposite (receiving the JS but not the CSS) is technically possible, if someone use the CSS editor on the file you changed. But unless extremely critical, I don't consider it either.